test/e2e/cypress/integration/search.spec.js

```diff
@@ -11,6 +11,7 @@ describe('Search', function() {
       properties: {
         firstName: {
           type: 'text',
+          fielddata: true,
```

Comment:
shouldn't this be camel-cased?

Reply:
No it's an elasticsearch prop: https://www.elastic.co/guide/en/elasticsearch/reference/current/fielddata.html